postgres/pyproject.toml

```diff
@@ -37,11 +37,14 @@ license = "BSD-3-Clause"
 
 [project.optional-dependencies]
 deps = [
-    "azure-identity==1.14.0; python_version > '3.0'",
-    "boto3==1.28.55; python_version > '3.0'",
-    "cachetools==5.3.1; python_version > '3.0'",
-    "psycopg2-binary==2.9.8; python_version > '3.0'",
-    "semver==3.0.1; python_version > '3.0'",
+    "azure-identity==1.15.0; python_version > '3.0'",
+    "boto3==1.17.112; python_version < '3.0'",
```

Comment:
`postgres` has dropped support for python2. Shall we remove python2 deps?

Reply:
Thanks, will do. Created a card for us to tweak the depedency update script to avoid this in the future!